tokio/tests/macros_select.rs

```diff
@@ -207,6 +207,7 @@ async fn nested() {
 }
 
 #[maybe_tokio_test]
+#[cfg(target_pointer_width = "64")]
```

Comment:
as with join, could we make a separate set of assertions about the future size on 32-bit platforms?